src/components/NotFoundPage.css

```diff
@@ -0,0 +1,31 @@
+.not-found-container {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	height: 100vh;
+	width: 100%;
+}
+
+.not-found-heading {
+	font-size: 4.5em;
+	margin-bottom: 0;
+}
+
+.Nav-link {
+	--color-text: var(--color-cobalt-blue);
+	color: var(--color-text);
+	font-size: 1.4em;
```

Comment:
While I appreciate larger icons and text for increased accessibility, these almost feel inconsistent with the rest of the design. Is the plan to increase the text and button size throughout the app? If so, these work nicely. However, I wanted to point out that in mobile view, the large icons are causing for part of the text to be hidden when you click to the `List` page. Take a look: 


https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/97368143/ac14cc23-7528-43fd-96d9-4a7db91fab74

Perhaps consider reducing the sizing slightly or adjusting the margins.

Reply:
Hi @polly89, I agree with your recommendation! I have changed the font-size from 1.4em to 0.85em, and it aligns better with our styling.

Reply:
These look great now.